netebpfext/net_ebpf_ext.c

```diff
@@ -352,7 +403,11 @@ net_ebpf_extension_add_wfp_filters(
         filter.action.calloutKey = *filter_parameter->callout_guid;
         filter.filterCondition = (FWPM_FILTER_CONDITION*)conditions;
         filter.numFilterConditions = condition_count;
-        filter.subLayerKey = EBPF_SUBLAYER;
+        if (filter_parameter->sublayer_guid) {
```

Comment:
nit: for pointers can we use NULL check? I dont know if there is any coding guidance around this.

Reply:
updated